oak_functions/loader/Cargo.toml

```diff
@@ -7,6 +7,14 @@ authors = ["Razieh Behjati <razieh@google.com>"]
 edition = "2018"
 license = "Apache-2.0"
 
+[lib]
+name = "oak_functions_loader"
+path = "src/lib.rs"
```

Comment:
Do these paths need to be specified? AFAICT they are the default values: https://doc.rust-lang.org/cargo/guide/project-layout.html